sdk/src/main/java/io/radar/sdk/RadarLocationManager.kt

```diff
@@ -202,12 +202,13 @@ internal class RadarLocationManager(
         }
 
         if (tracking) {
-            val foregroundService = RadarSettings.getForegroundService(context)
-            if (foregroundService != null && options.foregroundServiceEnabled) {
+            if (options.foregroundServiceEnabled) {
+                val foregroundService = RadarSettings.getForegroundService(context)
+                    ?: RadarTrackingOptions.RadarTrackingOptionsForegroundService()
                 if (!foregroundService.updatesOnly) {
                     this.startForegroundService(foregroundService)
                 }
-            } else if (RadarForegroundService.started) {
+            } else {
                 this.stopForegroundService()
             }
```

Comment:
This is the fix for the missing "default" notification. The other changes are to limit changes to the "started" attribute on the service.